helm/values.yaml

```diff
@@ -45,22 +59,23 @@ secret:
 
 # appSettings:
 #   env:
-#    FIFTYONE_DATABASE_ADMIN is set to `false` by default for v1.5.7 installs
-#     If you are performing a new install or an upgrade from v1.0 or earlier you may want to set
-#     this value to `true`.
-#     Please see https://helm.fiftyone.ai/#initial-installation-vs-upgrades for details
-#    FIFTYONE_DATABASE_ADMIN: false
-#    Set FIFTYONE_PLUGINS_DIR and FIFTYONE_PLUGINS_CACHE_ENABLED if you are
-#     enabling plugins in the `fiftyone-app` deployment
-#    See https://github.com/voxel51/fiftyone-teams-app-deploy/tree/main/helm#enabling-fiftyone-teams-plugins
-#    FIFTYONE_PLUGINS_DIR: /opt/plugins
+#     # FIFTYONE_DATABASE_ADMIN is set to `false` by default for installs.
+#     # If you are performing a new install or an upgrade from
+#     # v1.0 or earlier you may want to set this value to `true`.
+#     # Please see https://helm.fiftyone.ai/#initial-installation-vs-upgrades for details.
+#     # FIFTYONE_DATABASE_ADMIN: false
+#     FIFTYONE_DATABASE_ADMIN: true
```

Comment:
should we provide both values for all true/false options?

Reply:
I believe yes. Instead of asking the user to change the value, they can just comment and uncomment a line.  CS has mentioned that customers need reminding of this and this makes it easier.